apteryxd.c

```diff
@@ -322,6 +323,8 @@ call_refreshers (const char *path)
         rpc_client rpc_client;
         rpc_message_t msg = {};
 
+        pthread_mutex_lock (&refresh_lock);
```

Comment:
Dont we want this to be on the actual refresher itself rather than globally stopping more than one refresher being called at once?

Reply:
Yeah, I had thought that was a possibility. I was a bit split - if you think that's worth it I'll make that change.